.mage/js.go

```diff
@@ -229,8 +250,13 @@ func (js Js) ServeMain() error {
 
 // Messages extracts the frontend messages via babel.
 func (js Js) Messages() error {
+	isCI := os.Getenv("CI") == "true"
```

Comment:
This should not be necessary

Reply:
It shouldn't, but it is unfortunately. When the Github Action runner checks out the repository, it will result in all files being modtimed to the current time. Hence, all mage target checks will return true. The only way to still make use of caching is to use a different target dependency logic for the CI environment.